### DISCUSSION TITLE
Add a Web interface to present the project status

### DISCUSSION THREAD

**Discussion by cadayton** [jdhitsolutions/PSProjectStatus#3] [Ideas]
![image](https://user-images.githubusercontent.com/5587738/159018663-cca17b8f-0fb6-4cf6-989d-547d78449b54.png)

Great idea that thanks for sharing.
My first thoughts are maybe as well as presenting the project data in a CLI grid, create a Web table using the module 'Pode.Web' to present and maintain the data.  The above image shows doing the same with the [psConsole](https://cadayton.keybase.pub/PSGallery/Scripts/psConsole.html) utility that I wrote.

Perhaps I put together a quick demo using your project that can be expanded upon.

**Comment by cadayton** [comment]
![image](https://user-images.githubusercontent.com/5587738/159022189-40f963af-ab58-49dd-ae3d-f04692cf7881.png)

Another example.

**Comment by jdhitsolutions** [comment]
I wouldn't have thought of this. Thanks.

**Comment by cadayton** [comment]
Here is a quick hack at a web interface using Pode.Web
![image](https://user-images.githubusercontent.com/5587738/159084613-68fbfd66-d2aa-411c-96a5-f0639f620771.png)

[Code for generating the page](https://cadayton.keybase.pub/PSGallery/Pode/PSProjectStatus_page.ps1)
